src/applications/personalization/dashboard/tests/e2e/benefit-applications-in-progress.cypress.spec.js

```diff
@@ -126,135 +125,61 @@ describe('The My VA Dashboard', () => {
     },
   ];
 
-  context('when myVaFormSubmissionStatuses feature toggle is on', () => {
+  beforeEach(() => {
+    cy.intercept('/v0/profile/service_history', serviceHistory);
+    cy.intercept('/v0/profile/full_name', fullName);
+    cy.intercept(
+      '/v0/disability_compensation_form/rating_info',
+      disabilityRating,
+    );
+    cy.intercept('/v0/profile/payment_history', paymentHistory);
+  });
```

Comment:
We should probably leave the `/v0/feature_toggles*` mock in here since the page still makes this network call, but it can just return an empty response.  I've run into a few "flakiness" issues with un-mocked api requests lately (usually with other endpoints) but this can also clean up our cypress error log, even if it works fine without the mock:
```
      cy.intercept('GET', '/v0/feature_toggles*', {
        data: {
          features: [],
        },
      });
```